Ansible/roles/ldap/files/etc/openldap/slapd.d/orgs.ldif

```diff
@@ -4556,6 +4556,212 @@ o: UDSM
 description: University of Dar es Salaam
 registeredAddress: https://udsm.ac.tz/
 
+# DIT, TZ, Organisations, local
+dn: o=DIT,c=TZ,ou=Organisations,dc=local
+objectClass: organization
+objectClass: top
+o: DIT
+description: Dar es Salaam Institute of Technology
+registeredAddress: http://www.dit.ac.tz/
+
+# MZU, TZ, Organisations, local
+dn: o=MZU,c=TZ,ou=Organisations,dc=local
+objectClass: organization
+objectClass: top
+o: MZU
+description: Mzumbe University
+registeredAddress: http://mzumbe.ac.tz/
+
+
+# OUT, TZ, Organisations, local
+dn: o=OUT,c=TZ,ou=Organisations,dc=local
+objectClass: organization
+objectClass: top
+o: OUT
+description: Open University of Tanzania
+registeredAddress: http://out.ac.tz/
+
+
+# HKMU, TZ, Organisations, local
+dn: o=HKMU,c=TZ,ou=Organisations,dc=local
+objectClass: organization
+objectClass: top
+o: HKMU
+description: Hubert Kairuki Memorial University
+registeredAddress: http://www.hkmu.ac.tz/
+
+# SUA, TZ, Organisations, local
+dn: o=SUA,c=TZ,ou=Organisations,dc=local
+objectClass: organization
+objectClass: top
+o: SUA
+description: Sokoine University of Agriculture
+registeredAddress: http://www.suanet.ac.tz/
+
+# IMTU, TZ, Organisations, local
+dn: o=IMTU,c=TZ,ou=Organisations,dc=local
+objectClass: organization
+objectClass: top
+o: IMTU
+description: Int. Medical and Technological University
+registeredAddress: http://imtu.ac.tz/
+
+# TU, TZ, Organisations, local
+dn: o=TU,c=TZ,ou=Organisations,dc=local
+objectClass: organization
+objectClass: top
+o: TU
+description: Tumaini University Makumira
+registeredAddress: http://www.makumira.ac.tz/
+
+# SUZA, TZ, Organisations, local
+dn: o=SUZA,c=TZ,ou=Organisations,dc=local
+objectClass: organization
+objectClass: top
+o: SUZA
+description: State University of Zanzibar
+registeredAddress: http://www.suza.ac.tz/
+
+
+# SJUT, TZ, Organisations, local
+dn: o=SJUT,c=TZ,ou=Organisations,dc=local
+objectClass: organization
+objectClass: top
+o: SJUT
+description: St. John's University of Tanzania
```

Comment:
I'm not sure if there is a problem with including the ' in the name here - it might not be interpreted correctly. @fmarco76 could you please comment ?